alpha/apps/kaltura/lib/rules/kRule.php

```diff
@@ -144,6 +150,7 @@ protected function fulfilled()
 				KalturaLog::debug("Condition [" . get_class($condition) . "] not  fulfilled");
 				return false;
 			}
+			$this->extraProperties = array_merge($this->getExtraProperties(), $condition->getExtraProperties());
```

Comment:
In case the first condition is serveFromKes and KES is down you won't go to running the second IP condition and wont fill in the extraProperties data.